source/change-streams/tests/change-streams.json

```diff
@@ -675,6 +675,123 @@
           }
         ]
       }
+    },
+    {
+      "description": "Test consecutive resume",
+      "minServerVersion": "3.6.0",
```

Comment:
I think the JSON file needs to be regenerated. This is out of sync with the YAML below.

Reply:
Yeah, it was sitting un-pushed in my local repo.  Done.